### DISCUSSION TITLE
Pdfs which do not need OCR

### DISCUSSION THREAD

**Discussion by milossramek** [ciur/papermerge#318] [Q&A]
Hi,
In the videos only scanned documents are mentioned, which are OCR-ed on upload. What happens if I upload a pdf which is not a scan and thus does not need to be OCD-er? Is it recognized as such?
Thanks, Milos

**Comment by ciur** [comment]
No. Document will be OCR-ed anyway for the purpose of indexing. 
You always have original file available, thus you don't lose "overlay text" of the original PDF document.

**Reply by milossramek** [reply]
Thank you for the answer. To check, what happens with the pdfs I've uploaded and subsequently downloaded two documents to and from Papermerge - a scanned one had a layer with scanned text added, a second one with readable text was unchanged. Makes sense.
